packages/flutter_tools/lib/src/base/file_system.dart

```diff
@@ -103,6 +103,7 @@ class FileSystemUtils {
       final String name = '${baseName}_${i.toString().padLeft(2, '0')}.$ext';
       final File file = fs.file(_fileSystem.path.join(dir.path, name));
       if (!file.existsSync()) {
+        file.createSync(recursive: true);
```

Comment:
Other callers (were there any?) of `getUniqueFile()` were probably not expecting the file to be created by the call. Do you need to change any of the callsites?

Do you need to update any unit tests for `getUniqueFile()` for the change in behavior?